hipify-clang/CMakeLists.txt

```diff
@@ -114,6 +114,35 @@ install(
   PATTERN "openmp_wrappers" EXCLUDE
   )
 
+# Package: hipify-clang
+if (UNIX)
+    set(BUILD_DIR ${CMAKE_CURRENT_BINARY_DIR}/packages/hipify-clang)
+    configure_file(packaging/hipify-clang.txt ${BUILD_DIR}/CMakeLists.txt @ONLY)
+    add_custom_target(pkg_hipify-clang  COMMAND ${CMAKE_COMMAND} .
+        COMMAND rm -rf *.deb *.rpm *.tar.gz
+        COMMAND make package
+        COMMAND cp *.deb ${PROJECT_BINARY_DIR}
+        COMMAND cp *.rpm ${PROJECT_BINARY_DIR}
+        COMMAND cp *.tar.gz ${PROJECT_BINARY_DIR}
+        WORKING_DIRECTORY ${BUILD_DIR})
+
+    if(POLICY CMP0037)
+        cmake_policy(PUSH)
+        cmake_policy(SET CMP0037 OLD)
+    endif()
```

Comment:
I believe we follow the new cmake policy and do not need excessive warnings ([CMP0037](https://cmake.org/cmake/help/v3.11/policy/CMP0037.html)).